app.json

```diff
@@ -40,14 +40,14 @@
     {
       "plan": "heroku-postgresql:hobby-dev",
       "options": {
-        "version": "9.5"
+        "version": "9.6"
       },
       "as": "DATABASE"
     },
     {
       "plan": "heroku-redis:hobby-dev",
       "options": {
-        "version": "3.2"
+        "version": "6"
```

Comment:
These were the closest available versions for hobby-dev.

Reply:
you can update to the last version, all plugins.

Reply:
Including Postgres.